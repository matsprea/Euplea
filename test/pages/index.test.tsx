import React from 'react'
import { render, act } from '../testUtils'
import Home from '../../pages/index'

describe('Home page', () => {
  it('matches snapshot', async () => {
    await act(async () => {
      const { asFragment } = await render(<Home />, {})

      expect(asFragment()).toMatchSnapshot()
    })
  })

  // it('clicking button triggers alert', () => {
  //   const { getByText } = render(<Home />, {})
  //   window.alert = jest.fn()
  //   fireEvent.click(getByText('Test Button'))
  //   expect(window.alert).toHaveBeenCalledWith('With typescript and Jest')
  // })
})
