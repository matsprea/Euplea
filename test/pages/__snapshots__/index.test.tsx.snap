// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-k008qs"
  >
    <div
      class="css-1d4d9pm"
    >
      <nav
        class="css-eq3dzn"
      >
        <button
          aria-label="menu"
          class="chakra-button css-1vanx0x"
          type="button"
        >
          <svg
            aria-hidden="true"
            fill="none"
            focusable="false"
            height="1em"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <line
              x1="3"
              x2="21"
              y1="12"
              y2="12"
            />
            <line
              x1="3"
              x2="21"
              y1="6"
              y2="6"
            />
            <line
              x1="3"
              x2="21"
              y1="18"
              y2="18"
            />
          </svg>
        </button>
        <div
          class="css-mxserw"
        >
          <a
            class="chakra-link css-m909vr"
            href="/map"
          >
            <button
              aria-controls="menu-list-2"
              aria-expanded="false"
              aria-haspopup="menu"
              class="chakra-menu__menu-button css-kms2la"
              id="menu-button-2"
              variant="solid"
            >
              <span
                class="css-xl71ch"
              >
                <div
                  class="css-k008qs"
                >
                  <svg
                    class="chakra-icon css-5xffi5"
                    fill="none"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <polygon
                      points="1 6 1 22 8 18 16 22 23 18 23 2 16 6 8 2 1 6"
                    />
                    <line
                      x1="8"
                      x2="8"
                      y1="2"
                      y2="18"
                    />
                    <line
                      x1="16"
                      x2="16"
                      y1="6"
                      y2="22"
                    />
                  </svg>
                  <p
                    class="chakra-text css-1vzyiqb"
                  >
                    Mappa
                  </p>
                </div>
              </span>
            </button>
          </a>
          <div
            class="css-r6z5ec"
            data-popper-escaped=""
            data-popper-placement="right"
            data-popper-reference-hidden=""
            style="visibility: hidden; position: absolute; min-width: max-content; inset: 0 auto auto 0; left: 0px; top: 0px; margin: 0px; transform: translate(8px, 0px); --popper-transform-origin: left center;"
          >
            <div
              aria-orientation="vertical"
              class="chakra-menu__menu-list css-1oubb6u"
              id="menu-list-2"
              role="menu"
              style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.8) translateZ(0);"
              tabindex="-1"
            >
              <div
                class="css-1xxdnl4"
              />
              <div
                class="css-3k1d5n"
              >
                <svg
                  class="chakra-icon css-rmmnqe"
                  fill="none"
                  focusable="false"
                  height="1em"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <polygon
                    points="1 6 1 22 8 18 16 22 23 18 23 2 16 6 8 2 1 6"
                  />
                  <line
                    x1="8"
                    x2="8"
                    y1="2"
                    y2="18"
                  />
                  <line
                    x1="16"
                    x2="16"
                    y1="6"
                    y2="22"
                  />
                </svg>
                <h2
                  class="chakra-heading css-113ky19"
                >
                  Mappa
                </h2>
                <p
                  class="chakra-text css-0"
                >
                  This is the description for the dashboard.
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="css-mxserw"
        >
          <a
            class="chakra-link css-m909vr"
          >
            <button
              aria-controls="menu-list-4"
              aria-expanded="false"
              aria-haspopup="menu"
              class="chakra-menu__menu-button css-kms2la"
              id="menu-button-4"
              variant="outline"
            >
              <span
                class="css-xl71ch"
              >
                <div
                  class="css-k008qs"
                >
                  <svg
                    class="chakra-icon css-5xffi5"
                    fill="none"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <rect
                      height="18"
                      rx="2"
                      ry="2"
                      width="18"
                      x="3"
                      y="4"
                    />
                    <line
                      x1="16"
                      x2="16"
                      y1="2"
                      y2="6"
                    />
                    <line
                      x1="8"
                      x2="8"
                      y1="2"
                      y2="6"
                    />
                    <line
                      x1="3"
                      x2="21"
                      y1="10"
                      y2="10"
                    />
                  </svg>
                  <p
                    class="chakra-text css-1vzyiqb"
                  >
                    Calendar
                  </p>
                </div>
              </span>
            </button>
          </a>
          <div
            class="css-r6z5ec"
            data-popper-escaped=""
            data-popper-placement="right"
            data-popper-reference-hidden=""
            style="visibility: hidden; position: absolute; min-width: max-content; inset: 0 auto auto 0; left: 0px; top: 0px; margin: 0px; transform: translate(8px, 0px); --popper-transform-origin: left center;"
          >
            <div
              aria-orientation="vertical"
              class="chakra-menu__menu-list css-1oubb6u"
              id="menu-list-4"
              role="menu"
              style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.8) translateZ(0);"
              tabindex="-1"
            >
              <div
                class="css-1xxdnl4"
              />
              <div
                class="css-3k1d5n"
              >
                <svg
                  class="chakra-icon css-rmmnqe"
                  fill="none"
                  focusable="false"
                  height="1em"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <rect
                    height="18"
                    rx="2"
                    ry="2"
                    width="18"
                    x="3"
                    y="4"
                  />
                  <line
                    x1="16"
                    x2="16"
                    y1="2"
                    y2="6"
                  />
                  <line
                    x1="8"
                    x2="8"
                    y1="2"
                    y2="6"
                  />
                  <line
                    x1="3"
                    x2="21"
                    y1="10"
                    y2="10"
                  />
                </svg>
                <h2
                  class="chakra-heading css-113ky19"
                >
                  Calendar
                </h2>
                <p
                  class="chakra-text css-0"
                >
                  Calendar
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="css-mxserw"
        >
          <a
            class="chakra-link css-m909vr"
          >
            <button
              aria-controls="menu-list-6"
              aria-expanded="false"
              aria-haspopup="menu"
              class="chakra-menu__menu-button css-kms2la"
              id="menu-button-6"
              variant="solid"
            >
              <span
                class="css-xl71ch"
              >
                <div
                  class="css-k008qs"
                >
                  <svg
                    class="chakra-icon css-5xffi5"
                    fill="none"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M20 21v-2a4 4 0 0 0-4-4H8a4 4 0 0 0-4 4v2"
                    />
                    <circle
                      cx="12"
                      cy="7"
                      r="4"
                    />
                  </svg>
                  <p
                    class="chakra-text css-1vzyiqb"
                  >
                    Clients
                  </p>
                </div>
              </span>
            </button>
          </a>
          <div
            class="css-r6z5ec"
            data-popper-escaped=""
            data-popper-placement="right"
            data-popper-reference-hidden=""
            style="visibility: hidden; position: absolute; min-width: max-content; inset: 0 auto auto 0; left: 0px; top: 0px; margin: 0px; transform: translate(8px, 0px); --popper-transform-origin: left center;"
          >
            <div
              aria-orientation="vertical"
              class="chakra-menu__menu-list css-1oubb6u"
              id="menu-list-6"
              role="menu"
              style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.8) translateZ(0);"
              tabindex="-1"
            >
              <div
                class="css-1xxdnl4"
              />
              <div
                class="css-3k1d5n"
              >
                <svg
                  class="chakra-icon css-rmmnqe"
                  fill="none"
                  focusable="false"
                  height="1em"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M20 21v-2a4 4 0 0 0-4-4H8a4 4 0 0 0-4 4v2"
                  />
                  <circle
                    cx="12"
                    cy="7"
                    r="4"
                  />
                </svg>
                <h2
                  class="chakra-heading css-113ky19"
                >
                  Clients
                </h2>
                <p
                  class="chakra-text css-0"
                >
                  Calendar
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="css-mxserw"
        >
          <a
            class="chakra-link css-m909vr"
          >
            <button
              aria-controls="menu-list-8"
              aria-expanded="false"
              aria-haspopup="menu"
              class="chakra-menu__menu-button css-kms2la"
              id="menu-button-8"
              variant="solid"
            >
              <span
                class="css-xl71ch"
              >
                <div
                  class="css-k008qs"
                >
                  <svg
                    class="chakra-icon css-5xffi5"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M457.74 170.1a30.26 30.26 0 00-11.16-2.1h-.4c-20.17.3-42.79 19.19-54.66 47.76-14.23 34.18-7.68 69.15 14.74 78.14a30.21 30.21 0 0011.15 2.1c20.27 0 43.2-19 55.17-47.76 14.13-34.18 7.48-69.15-14.84-78.14zM327.6 303.48C299.8 257.35 287.8 240 256 240s-43.9 17.46-71.7 63.48c-23.8 39.36-71.9 42.64-83.9 76.07a50.91 50.91 0 00-3.6 19.25c0 27.19 20.8 49.2 46.4 49.2 31.8 0 75.1-25.39 112.9-25.39S337 448 368.8 448c25.6 0 46.3-22 46.3-49.2a51 51 0 00-3.7-19.25c-12-33.55-60-36.71-83.8-76.07zM192.51 196a26.53 26.53 0 004-.3c23.21-3.37 37.7-35.53 32.44-71.85C224 89.61 203.22 64 181.49 64a26.53 26.53 0 00-4 .3c-23.21 3.37-37.7 35.53-32.44 71.85C150 170.29 170.78 196 192.51 196zm174.41-59.85c5.26-36.32-9.23-68.48-32.44-71.85a26.53 26.53 0 00-4-.3c-21.73 0-42.47 25.61-47.43 59.85-5.26 36.32 9.23 68.48 32.44 71.85a26.53 26.53 0 004 .3c21.73 0 42.51-25.71 47.43-59.85zM105.77 293.9c22.39-9 28.93-44 14.72-78.14C108.53 187 85.62 168 65.38 168a30.21 30.21 0 00-11.15 2.1c-22.39 9-28.93 44-14.72 78.14C51.47 277 74.38 296 94.62 296a30.21 30.21 0 0011.15-2.1z"
                      fill="none"
                      stroke-miterlimit="10"
                      stroke-width="32"
                    />
                  </svg>
                  <p
                    class="chakra-text css-1vzyiqb"
                  >
                    Animals
                  </p>
                </div>
              </span>
            </button>
          </a>
          <div
            class="css-r6z5ec"
            data-popper-escaped=""
            data-popper-placement="right"
            data-popper-reference-hidden=""
            style="visibility: hidden; position: absolute; min-width: max-content; inset: 0 auto auto 0; left: 0px; top: 0px; margin: 0px; transform: translate(8px, 0px); --popper-transform-origin: left center;"
          >
            <div
              aria-orientation="vertical"
              class="chakra-menu__menu-list css-1oubb6u"
              id="menu-list-8"
              role="menu"
              style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.8) translateZ(0);"
              tabindex="-1"
            >
              <div
                class="css-1xxdnl4"
              />
              <div
                class="css-3k1d5n"
              >
                <svg
                  class="chakra-icon css-rmmnqe"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 512 512"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M457.74 170.1a30.26 30.26 0 00-11.16-2.1h-.4c-20.17.3-42.79 19.19-54.66 47.76-14.23 34.18-7.68 69.15 14.74 78.14a30.21 30.21 0 0011.15 2.1c20.27 0 43.2-19 55.17-47.76 14.13-34.18 7.48-69.15-14.84-78.14zM327.6 303.48C299.8 257.35 287.8 240 256 240s-43.9 17.46-71.7 63.48c-23.8 39.36-71.9 42.64-83.9 76.07a50.91 50.91 0 00-3.6 19.25c0 27.19 20.8 49.2 46.4 49.2 31.8 0 75.1-25.39 112.9-25.39S337 448 368.8 448c25.6 0 46.3-22 46.3-49.2a51 51 0 00-3.7-19.25c-12-33.55-60-36.71-83.8-76.07zM192.51 196a26.53 26.53 0 004-.3c23.21-3.37 37.7-35.53 32.44-71.85C224 89.61 203.22 64 181.49 64a26.53 26.53 0 00-4 .3c-23.21 3.37-37.7 35.53-32.44 71.85C150 170.29 170.78 196 192.51 196zm174.41-59.85c5.26-36.32-9.23-68.48-32.44-71.85a26.53 26.53 0 00-4-.3c-21.73 0-42.47 25.61-47.43 59.85-5.26 36.32 9.23 68.48 32.44 71.85a26.53 26.53 0 004 .3c21.73 0 42.51-25.71 47.43-59.85zM105.77 293.9c22.39-9 28.93-44 14.72-78.14C108.53 187 85.62 168 65.38 168a30.21 30.21 0 00-11.15 2.1c-22.39 9-28.93 44-14.72 78.14C51.47 277 74.38 296 94.62 296a30.21 30.21 0 0011.15-2.1z"
                    fill="none"
                    stroke-miterlimit="10"
                    stroke-width="32"
                  />
                </svg>
                <h2
                  class="chakra-heading css-113ky19"
                >
                  Animals
                </h2>
                <p
                  class="chakra-text css-0"
                >
                  Calendar
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="css-mxserw"
        >
          <a
            class="chakra-link css-m909vr"
          >
            <button
              aria-controls="menu-list-10"
              aria-expanded="false"
              aria-haspopup="menu"
              class="chakra-menu__menu-button css-kms2la"
              id="menu-button-10"
              variant="solid"
            >
              <span
                class="css-xl71ch"
              >
                <div
                  class="css-k008qs"
                >
                  <svg
                    class="chakra-icon css-5xffi5"
                    fill="none"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <line
                      x1="12"
                      x2="12"
                      y1="1"
                      y2="23"
                    />
                    <path
                      d="M17 5H9.5a3.5 3.5 0 0 0 0 7h5a3.5 3.5 0 0 1 0 7H6"
                    />
                  </svg>
                  <p
                    class="chakra-text css-1vzyiqb"
                  >
                    Stocks
                  </p>
                </div>
              </span>
            </button>
          </a>
          <div
            class="css-r6z5ec"
            data-popper-escaped=""
            data-popper-placement="right"
            data-popper-reference-hidden=""
            style="visibility: hidden; position: absolute; min-width: max-content; inset: 0 auto auto 0; left: 0px; top: 0px; margin: 0px; transform: translate(8px, 0px); --popper-transform-origin: left center;"
          >
            <div
              aria-orientation="vertical"
              class="chakra-menu__menu-list css-1oubb6u"
              id="menu-list-10"
              role="menu"
              style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.8) translateZ(0);"
              tabindex="-1"
            >
              <div
                class="css-1xxdnl4"
              />
              <div
                class="css-3k1d5n"
              >
                <svg
                  class="chakra-icon css-rmmnqe"
                  fill="none"
                  focusable="false"
                  height="1em"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <line
                    x1="12"
                    x2="12"
                    y1="1"
                    y2="23"
                  />
                  <path
                    d="M17 5H9.5a3.5 3.5 0 0 0 0 7h5a3.5 3.5 0 0 1 0 7H6"
                  />
                </svg>
                <h2
                  class="chakra-heading css-113ky19"
                >
                  Stocks
                </h2>
                <p
                  class="chakra-text css-0"
                >
                  Calendar
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="css-mxserw"
        >
          <a
            class="chakra-link css-m909vr"
          >
            <button
              aria-controls="menu-list-12"
              aria-expanded="false"
              aria-haspopup="menu"
              class="chakra-menu__menu-button css-kms2la"
              id="menu-button-12"
              variant="solid"
            >
              <span
                class="css-xl71ch"
              >
                <div
                  class="css-k008qs"
                >
                  <svg
                    class="chakra-icon css-5xffi5"
                    fill="none"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <rect
                      height="14"
                      rx="2"
                      ry="2"
                      width="20"
                      x="2"
                      y="7"
                    />
                    <path
                      d="M16 21V5a2 2 0 0 0-2-2h-4a2 2 0 0 0-2 2v16"
                    />
                  </svg>
                  <p
                    class="chakra-text css-1vzyiqb"
                  >
                    Reports
                  </p>
                </div>
              </span>
            </button>
          </a>
          <div
            class="css-r6z5ec"
            data-popper-escaped=""
            data-popper-placement="right"
            data-popper-reference-hidden=""
            style="visibility: hidden; position: absolute; min-width: max-content; inset: 0 auto auto 0; left: 0px; top: 0px; margin: 0px; transform: translate(8px, 0px); --popper-transform-origin: left center;"
          >
            <div
              aria-orientation="vertical"
              class="chakra-menu__menu-list css-1oubb6u"
              id="menu-list-12"
              role="menu"
              style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.8) translateZ(0);"
              tabindex="-1"
            >
              <div
                class="css-1xxdnl4"
              />
              <div
                class="css-3k1d5n"
              >
                <svg
                  class="chakra-icon css-rmmnqe"
                  fill="none"
                  focusable="false"
                  height="1em"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <rect
                    height="14"
                    rx="2"
                    ry="2"
                    width="20"
                    x="2"
                    y="7"
                  />
                  <path
                    d="M16 21V5a2 2 0 0 0-2-2h-4a2 2 0 0 0-2 2v16"
                  />
                </svg>
                <h2
                  class="chakra-heading css-113ky19"
                >
                  Reports
                </h2>
                <p
                  class="chakra-text css-0"
                >
                  Calendar
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="css-mxserw"
        >
          <a
            class="chakra-link css-m909vr"
          >
            <button
              aria-controls="menu-list-14"
              aria-expanded="false"
              aria-haspopup="menu"
              class="chakra-menu__menu-button css-kms2la"
              id="menu-button-14"
              variant="solid"
            >
              <span
                class="css-xl71ch"
              >
                <div
                  class="css-k008qs"
                >
                  <svg
                    class="chakra-icon css-5xffi5"
                    fill="none"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <circle
                      cx="12"
                      cy="12"
                      r="3"
                    />
                    <path
                      d="M19.4 15a1.65 1.65 0 0 0 .33 1.82l.06.06a2 2 0 0 1 0 2.83 2 2 0 0 1-2.83 0l-.06-.06a1.65 1.65 0 0 0-1.82-.33 1.65 1.65 0 0 0-1 1.51V21a2 2 0 0 1-2 2 2 2 0 0 1-2-2v-.09A1.65 1.65 0 0 0 9 19.4a1.65 1.65 0 0 0-1.82.33l-.06.06a2 2 0 0 1-2.83 0 2 2 0 0 1 0-2.83l.06-.06a1.65 1.65 0 0 0 .33-1.82 1.65 1.65 0 0 0-1.51-1H3a2 2 0 0 1-2-2 2 2 0 0 1 2-2h.09A1.65 1.65 0 0 0 4.6 9a1.65 1.65 0 0 0-.33-1.82l-.06-.06a2 2 0 0 1 0-2.83 2 2 0 0 1 2.83 0l.06.06a1.65 1.65 0 0 0 1.82.33H9a1.65 1.65 0 0 0 1-1.51V3a2 2 0 0 1 2-2 2 2 0 0 1 2 2v.09a1.65 1.65 0 0 0 1 1.51 1.65 1.65 0 0 0 1.82-.33l.06-.06a2 2 0 0 1 2.83 0 2 2 0 0 1 0 2.83l-.06.06a1.65 1.65 0 0 0-.33 1.82V9a1.65 1.65 0 0 0 1.51 1H21a2 2 0 0 1 2 2 2 2 0 0 1-2 2h-.09a1.65 1.65 0 0 0-1.51 1z"
                    />
                  </svg>
                  <p
                    class="chakra-text css-1vzyiqb"
                  >
                    Settings
                  </p>
                </div>
              </span>
            </button>
          </a>
          <div
            class="css-r6z5ec"
            data-popper-escaped=""
            data-popper-placement="right"
            data-popper-reference-hidden=""
            style="visibility: hidden; position: absolute; min-width: max-content; inset: 0 auto auto 0; left: 0px; top: 0px; margin: 0px; transform: translate(8px, 0px); --popper-transform-origin: left center;"
          >
            <div
              aria-orientation="vertical"
              class="chakra-menu__menu-list css-1oubb6u"
              id="menu-list-14"
              role="menu"
              style="transform-origin: var(--popper-transform-origin); opacity: 0; visibility: hidden; transform: scale(0.8) translateZ(0);"
              tabindex="-1"
            >
              <div
                class="css-1xxdnl4"
              />
              <div
                class="css-3k1d5n"
              >
                <svg
                  class="chakra-icon css-rmmnqe"
                  fill="none"
                  focusable="false"
                  height="1em"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <circle
                    cx="12"
                    cy="12"
                    r="3"
                  />
                  <path
                    d="M19.4 15a1.65 1.65 0 0 0 .33 1.82l.06.06a2 2 0 0 1 0 2.83 2 2 0 0 1-2.83 0l-.06-.06a1.65 1.65 0 0 0-1.82-.33 1.65 1.65 0 0 0-1 1.51V21a2 2 0 0 1-2 2 2 2 0 0 1-2-2v-.09A1.65 1.65 0 0 0 9 19.4a1.65 1.65 0 0 0-1.82.33l-.06.06a2 2 0 0 1-2.83 0 2 2 0 0 1 0-2.83l.06-.06a1.65 1.65 0 0 0 .33-1.82 1.65 1.65 0 0 0-1.51-1H3a2 2 0 0 1-2-2 2 2 0 0 1 2-2h.09A1.65 1.65 0 0 0 4.6 9a1.65 1.65 0 0 0-.33-1.82l-.06-.06a2 2 0 0 1 0-2.83 2 2 0 0 1 2.83 0l.06.06a1.65 1.65 0 0 0 1.82.33H9a1.65 1.65 0 0 0 1-1.51V3a2 2 0 0 1 2-2 2 2 0 0 1 2 2v.09a1.65 1.65 0 0 0 1 1.51 1.65 1.65 0 0 0 1.82-.33l.06-.06a2 2 0 0 1 2.83 0 2 2 0 0 1 0 2.83l-.06.06a1.65 1.65 0 0 0-.33 1.82V9a1.65 1.65 0 0 0 1.51 1H21a2 2 0 0 1 2 2 2 2 0 0 1-2 2h-.09a1.65 1.65 0 0 0-1.51 1z"
                  />
                </svg>
                <h2
                  class="chakra-heading css-113ky19"
                >
                  Settings
                </h2>
                <p
                  class="chakra-text css-0"
                >
                  Calendar
                </p>
              </div>
            </div>
          </div>
        </div>
      </nav>
      <div
        class="css-17xejub"
      />
      <div
        class="css-0"
      >
        <div
          class="css-gmuwbf"
        >
          <button
            class="chakra-button css-1rwlt5g"
            type="button"
          >
            Cerca
          </button>
        </div>
      </div>
    </div>
    <div
      class="App  css-1ofqig9"
    >
      <div
        class="css-fi62t4"
      >
        <div
          class="css-u796g3"
        >
          <h1
            class="chakra-heading css-1dklj6k"
          >
            Euplea
          </h1>
        </div>
      </div>
      <div
        class="css-8atqhb"
      >
        <div
          class="chakra-skeleton css-iwltc"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
